docs/breaking-changes.md

```diff
@@ -14,6 +14,11 @@ This document uses the following convention to categorize breaking changes:
 
 ## Planned Breaking API Changes (10.0)
 
+### API Changed: `BrowserWindowConstructorOptions` is renamed to `BrowserWindowOptions`
+
+For TypeScript users, the type name of `BrowserWindow`'s options has changed
+form `BrowserWindowConstructorOptions` to `BrowserWindowOptions`.
```

Comment:
We don't typically list type changes as breaking changes to the API. I think we should continue that tradition for a few reasons:

1. We fairly regularly make relatively minor changes to types (e.g. refining types from `any` to a more specific type, or fixing incorrect types). Ensuring we including all such changes in the breaking changes notes would be a fairly significant undertaking and would result in a lot of new breaking change notes.
2. The TS compiler will automatically catch these changes and notify folks who are upgrading anyway.

That said, I could see an argument for having a single item under the "breaking API changes" for "here's what types changed: ..."

Reply:
This was requested by @malept, I don't have a strong feeling for documenting this.

Reply:
I've added this topic to the Ecosystem WG meeting agenda.